api/src/main/resources/org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml

```diff
@@ -14,38 +14,34 @@
     "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
     "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 <hibernate-mapping>
-	<class name="org.openmrs.OrderFrequency" table="order_frequency">
```

Comment:
Seems like this file has been formatted? Can you try undoing the changes in it? Then just add the single for the new field and see if that resolves it